docs/repositories.md

```diff
@@ -101,7 +101,7 @@ The repository metadata file is pushed to the registry using a special tag named
 
 ## CoreDNS plugins repositories
 
-CoreDNS plugins repositories are expected to be hosted in Github, Gitlab or Bitbucket repos. When adding your repository to Artifact Hub, the url used **must** follow the following format:
+CoreDNS plugins repositories are expected to be hosted in GitHub, Gitlab or Bitbucket repos. When adding your repository to Artifact Hub, the url used **must** follow the following format:
```

Comment:
GitLab is also a brand, but I'm going to skip it for now. (Happy to fix it if asked.)

Reply:
Yes, please do 👍